libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackHelper.cs

```diff
@@ -253,7 +259,7 @@ public static Activity EventToActivity(EventRequest eventRequest, SlackClientWra
                     activity.Text = message.Text;
                     if (message.AdditionalProperties.ContainsKey("files"))
                     {
-                        var attachments = new List<Attachment>();
+                        var attachments = new List<Schema.Attachment>();
```

Comment:
Note: This change and the one on line 289 were to disambiguate from `SlackAPI.Attachment`